docker_images/sklearn/app/main.py

```diff
@@ -38,7 +37,6 @@
 }
 
 
-@functools.lru_cache()
```

Comment:
this is removed since `get_pipeline` is always run with no args, and therefore the cache would return the wrong output.